cross-chain/arbitrum/deploy_l2/21_authorize_wormhole_gateway_as_minter.ts

```diff
@@ -0,0 +1,23 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { execute } = deployments
+  const { governance } = await getNamedAccounts()
+
+  const ArbitrumWormholeGateway = await deployments.get(
+    "ArbitrumWormholeGateway"
+  )
+
+  await execute(
+    "ArbitrumTBTC",
+    { from: governance, log: true, waitConfirmations: 1 },
+    "addMinter",
+    ArbitrumWormholeGateway.address
+  )
+}
+
+export default func
+
+func.tags = ["AuthorizeWormholeGateway"]
```

Comment:
I think we may need to force the order of this script by making it dependent on `ArbitrumTBTC` and `ArbitrumWormholeGateway` and using `runAtTheEnd = true`

Reply:
Same answer as in https://github.com/keep-network/tbtc-v2/pull/554#discussion_r1130049619

Reply:
And same response as in https://github.com/keep-network/tbtc-v2/pull/554#discussion_r1131029951